src/ducks/Alert/components/AlertStepsSelector/AlertStepDescription/AlertStepDescription.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import styles from './AlertStepDescription.module.scss'
+
+const DESCRIPTION_TYPES = {
+  TITLE: 'title'
+}
+
+function checkValueByType (values, type) {
+  switch (type) {
+    case 'Check name and description':
+      return {
+        value: values.title,
+        type: DESCRIPTION_TYPES.TITLE
+      }
+    case 'Name & Description':
+      return {
+        value: values.title,
+        type: DESCRIPTION_TYPES.TITLE
+      }
```

Comment:
Seems like these two cases have similar returns. If it is not subject to change, then it can be simplified to:
```js
case 'Check name and description':
case 'Name & Description':
   return ...
```